chalicelib/check_setup.json

```diff
@@ -110,7 +110,7 @@
         "title": "Change in item counts",
         "group": "Metadata checks",
         "schedule": {
-            "morning_checks": {
+            "morning_checks_2": {
```

Comment:
Why this rename?  Is this just a name or something semantic?

Reply:
`morning_checks_2` is actually a separate, unused schedule that is staggered from the regular `morning_checks`. You can see the schedule in `app.py`.